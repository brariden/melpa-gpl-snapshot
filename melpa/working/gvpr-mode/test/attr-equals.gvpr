N {
  if(attr_equals($,"x","bar")) {
    printf("NODE \"%s\" MATCHES.\n",$.name);
  } else {
    printf("NODE \"%s\" DOES NOT MATCH.\n",$.name);
  }
}
