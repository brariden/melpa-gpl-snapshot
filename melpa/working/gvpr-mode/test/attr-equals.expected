NODE "A" MATCHES.
NODE "B" MATCHES.
NODE "Z" DOES NOT MATCH.
NODE "Y" DOES NOT MATCH.
NODE "X" DOES NOT MATCH.
NODE "W" DOES NOT MATCH.
